accurest-core/src/main/groovy/io/codearte/accurest/dsl/internal/Request.groovy

```diff
@@ -100,6 +102,26 @@ class Request extends Common {
 		return body
 	}
 
+	void multipart(Map<String, Object> body) {
+		this.multipart = new Multipart(convertObjectsToDslProperties(body))
+	}
+
+	void multipart(List multipartAsList) {
+		this.multipart = new Multipart(convertObjectsToDslProperties(multipartAsList))
+	}
+
+	void multipart(DslProperty dslProperty) {
+		this.multipart = new Multipart(dslProperty)
+	}
+
+	void multipart(Object multipartAsValue) {
+		this.multipart = new Multipart(multipartAsValue)
+	}
+
+	Multipart getMultipart() {
```

Comment:
you don't need this method casue

`Multipart multipart` generates a getter and setter for you